tests/conftest.py

```diff
@@ -4618,3 +4619,70 @@ def finalizer():
 
     request.addfinalizer(finalizer)
     return factory
+
+
+@pytest.fixture(scope="session", autouse=True)
+def patch_consumer_toolbox_with_secret():
+    """
+    Patch the rook-ceph-tools deployment with ceph.admin key. Applicable for MS platform only to enable rook-ceph-tools
+    to run ceph commands until we have the fix for rook-ceph-tools in consumer cluster
+
+    """
+    # Get the secret from provider if MS multicluster run
+    if not (
+        config.multicluster
+        and config.ENV_DATA.get("platform", "").lower()
+        in constants.MANAGED_SERVICE_PLATFORMS
+        and not config.RUN["cli_params"].get("deploy")
+    ):
+        return
+
+    current_cluster = config.cluster_ctx
```

Comment:
Isn't context directly stored in this variable:
https://github.com/red-hat-storage/ocs-ci/blob/bcd66b7bfd3366ec3c4f67eba37183218daff0bb/ocs_ci/framework/__init__.py#L135

Reply:
Yes, correct. Made the change.